controllers/storageconfig_controller.go

```diff
@@ -91,11 +94,23 @@ func (r *StorageSecretReconciler) reconcileSecret(secret *corev1.Secret,
 	if err != nil {
 		if apierrs.IsNotFound(err) {
 			log.Info("No data connections found in namespace ", secret.Namespace)
+			return nil
 		}
```

Comment:
I would add an else condition to log the error (and return?): currently any error different from `not found` is ignored

Reply:
You are right. I added `else`.